.travis.yml

```diff
@@ -1,47 +1,72 @@
-
+sudo: false
+os:
+  - linux
+  - osx
 language: cpp
-
 compiler:
  - clang
 
 env:
- - GOPATH=${TRAVIS_BUILD_DIR}/go CPPFLAGS="-I/usr/src/gmock/include" CXXFLAGS="-Wall -Werror"
+  - CXXFLAGS="-std=c++11"
```

Comment:
Huh? I don't understand why some variables have moved to `before_install`, while `CXXFLAGS` remains here (with a rather peculiar value, since the `configure` script should be able to figure that out by itself?!?)...

I agree that I kind of abused `env`, I just wanted to set some environment variables, and `env` is more to add an additional dimension to the build matrix. Move this to an `export CXXFLAGS=...` in `before_install`?